MapboxCoreNavigation/TunnelInfo.swift

```diff
@@ -0,0 +1,14 @@
+
+import Foundation
+import MapboxNavigationNative
+
+/**
+ `TunnelInfo` is used for naming incoming tunnels, together with route alerts.
+ */
+public struct TunnelInfo {
+    public let name: String
```

Comment:
Since this struct stores only a name, do we need the struct, or can we inline the name?

If we do expect to extend this type with more information in the future, then consider calling it Tunnel. (Compare to MapboxDirections’ [Road](https://github.com/mapbox/mapbox-directions-swift/blob/4184df3c07694f75738a99728cdfa5504c8e4632/Sources/MapboxDirections/RouteStep.swift#L307) struct.)

Reply:
Situation here is same as for `BorderCrossingInfo`. I wanted to have similar configuration for each Route Alert possible. IMHO if we have some mix of `struct`s, `tuple`s and inlined values that would look inconsistent. That's why I decided to stick with `struct`s everywhere because this is the most universal approach and it allows extending functionality and data without breaking backwards compatibility.